biggerquery/user_commons/fastai/predict.py

```diff
@@ -0,0 +1,35 @@
+import apache_beam as beam
+from apache_beam.pvalue import AsSingleton
+
+
+def side_input_factory(p, item, label_name):
+    return AsSingleton(p
+                       | label_name + ', creating collection' >> beam.Create([item])
+                       | label_name + ', combining globally' >> beam.CombineGlobally(beam.combiners.ToListCombineFn()))
+
+
+class PredictOfferPotential(beam.DoFn):
```

Comment:
Maybe just `Predictor` instead of the `PredictOfferPotential`?

Reply:
done